dist/common/scripts/scylla_ntp_setup

```diff
@@ -78,7 +78,7 @@ if __name__ == '__main__':
             run('rc-service ntpd start')
 
     if is_redhat_variant():
-        if redhat_version() >= 8:
+        if not version_compare(redhat_version(), '8'):
```

Comment:
It will be clearer if you write 

    if parse_version(redhat_version()) >= parse_version('8'):

otherwise we have to check what version_compare means, we can't guess it's "less than".

Reply:
Fixed. thanks.